core/common/src/rpc/web/WebAppRpcRequest.ts

```diff
@@ -343,7 +343,7 @@ export class WebAppRpcRequest extends RpcRequest {
     const requestClass = this.supplyRequest();
     const fetchFunction = this.supplyFetch();
 
-    const path = new URL(this.path, typeof (location) !== "undefined" ? location.origin : undefined);
+    const path = new URL(this.path, typeof (location) !== "undefined" && location.origin !== "null" ? location.origin : undefined);
```

Comment:
When would `location.origin` ever be set to (the string) `"null"`?

Reply:
I added this line because in my system locally location was defined but it was set to the string "null". I don't know why this is, maybe because of jsdom-global doing so in node?

Regardless, because it was set to null this line was crashing,
and in the scorecard validator they were mocking it with the line:
`(global as any).location = { origin: undefined };`
https://dev.azure.com/bentleycs/beconnect/_git/BCSM?path=/ReleaseValidation/Validators/iModelJs_V2/src/RpcList.ts&version=GBmaster&line=18&lineEnd=19&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

My change removes the need to mock location origin as when it's null it just sets it to undefined.